src/App.vue

```diff
@@ -1,6 +1,3 @@
 <template>
   <router-view />
 </template>
-
-<style src="./assets/scss/styles.scss" lang="scss">
-</style>
```

Comment:
please revert this change, write only global styles inside this file without scss variables

Reply:
you can connect file with scss variables in your vue.config.js file by replacing import of styles.scss (global styles) file